lms/app.py

```diff
@@ -14,6 +14,7 @@ def create_app(global_config, **settings):  # pylint: disable=unused-argument
     config = configure(settings=settings)
 
     config.include("pyramid_jinja2")
+    config.include("pyramid_services")
```

Comment:
The `pyramid_services` package was already a dependency in `requirements.in` but the extension had been removed from the app itself. Add it back